testing/testing.h

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include <string>
+#include <sstream>
```

Comment:
So this was me trying to emulate googletest and allow things like this that would be printed on failure.

```cpp
ASSERT(false) << "Uh oh!";
```

I later realised doing this probably requires using exceptions to circumvent control flow so I abandoned it.

Having a `printf` like API is interesting, still think it would be tricky to do in a single statement but I might be missing something.